cms/templates/certificates.html

```diff
@@ -30,8 +32,16 @@
 
 <%block name="requirejs">
   require(["js/certificates/factories/certificates_page_factory"], function(CertificatesPageFactory) {
-      if(${escape_json_dumps(has_certificate_modes)}) {
-          CertificatesPageFactory(${escape_json_dumps(certificates) | n}, "${certificate_url}", "${course_outline_url}", ${escape_json_dumps(course_modes) | n}, ${escape_json_dumps(certificate_web_view_url) | n}, ${escape_json_dumps(is_active) | n}, ${escape_json_dumps(certificate_activation_handler_url) | n} );
+      if(${has_certificate_modes | n, dump_js_escaped_json}) {
```

Comment:
@andy-armstrong Could you answer the following:
1. Is it safe to drop everything if the conditional is false, including the `require()` line?
2. Do I do that with an `% if has_certificate_modes:` and `% endif` immediately inside the named block?  Other?
Thanks.